src/queries/incremental-compilation-in-detail.md

```diff
@@ -507,9 +507,10 @@ session. The overhead of doing so is a few percent of total compilation time.
 Data structures used as query results could be factored in a way that removes
 edges from the dependency graph. Especially "span" information is very volatile,
 so including it in query result will increase the chance that that result won't
-be reusable. See https://github.com/rust-lang/rust/issues/47389 for more
+be reusable. See [https://github.com/rust-lang/rust/issues/47389] for more
```

Comment:
You can use a shortcut link here I think so you don't duplicate the url below.
```suggestion
be reusable. See <https://github.com/rust-lang/rust/issues/47389> for more
```

Reply:
I didn't know angle brackets to turn URL into a link. Thanks!